bubble/src/main/java/pl/touk/android/bubble/Degree.kt

```diff
@@ -3,5 +3,6 @@ package pl.touk.android.bubble
 internal class Degree {
     companion object Degree {
         val MINUS_45 = (-Math.PI/4).toFloat()
+        val PLUS_45 = (Math.PI/4).toFloat()
```

Comment:
It could be simpler without class:
```kotlin
internal object Degree {
    val MINUS_45 = (-Math.PI / 4).toFloat()
    val PLUS_45 = (Math.PI / 4).toFloat()
}
```